README.md

```diff
@@ -1,145 +1,92 @@
 # Truss
 
-**Serve any model without boilerplate code**
-
-![Truss logo](https://raw.githubusercontent.com/basetenlabs/truss/main/docs/assets/truss_logo_horizontal.png)
+**The simplest way to serve AI/ML models in production**
 
 [![PyPI version](https://badge.fury.io/py/truss.svg)](https://badge.fury.io/py/truss)
 [![ci_status](https://github.com/basetenlabs/truss/actions/workflows/main.yml/badge.svg)](https://github.com/basetenlabs/truss/actions/workflows/main.yml)
 
-Meet Truss, a seamless bridge from model development to model delivery. Truss presents an open-source standard for packaging models built in any framework for sharing and deployment in any environment, local or production.
-
-Get started with the [end-to-end tutorial](https://truss.baseten.co/e2e).
-
-## What can I do with Truss?
-
-If you've ever tried to get a model out of a Jupyter notebook, Truss is for you.
+## Why Truss?
 
-Truss exposes just the right amount of complexity around things like Docker and APIs without you really having to think about them. Here are some of the things Truss does:
+* **Write once, deploy anywhere:** A model packaged with Truss can be deployed on supported providers without changing the underlying server implementation.
```

Comment:
it's without changing anything from the user perspective? I feel like we don't wanna say server implementation here/